### ISSUE TITLE
Add tigerlib templates to VS Code

### ISSUE THREAD

**Issue by npmanos** [itrt4176/rapidreact2022#3]
## Overview

Update setup scripts to copy tigerlib templates to WPILib VS Code extension template folder

## Notes

Windows location: %USERPROFILE%\.vscode\extensions\wpilibsuite.vscode-wpilib-2022.*\resources\java\src\commands\
Linux location : ~/.vscode/extensions/wpilibsuite.vscode-wpilib-2022.*/resources/java/src/commands/

**Comment by npmanos** [comment]
@limited Can you confirm if the folder's location on macOS is the same as linux? And if it's not, can you find it?

**Comment by limited** [comment]
Yes, can confirm this is the path on OS X 12.1 for the 2021 WPIlib. I assume its the same for 2022
`/Users/efriedrich/.vscode/extensions/wpilibsuite.vscode-wpilib-2021.1.2/resources/java/src/commands`

**Comment by npmanos** [comment]
Windows batch scripts don't appear to allow wildcards in the folder names. Putting this on the back burner until I have time to research a solution.
